gnucash/gnome-utils/gnc-dense-cal.c

```diff
@@ -976,16 +976,16 @@ gnc_dense_cal_draw_to_buffer(GncDenseCal *dcal)
     /* get the colors */
     {
          GdkRGBA color;
-         gchar *color_extension = NULL;
+         gchar *class_extension = NULL;
 
          gtk_style_context_get_color (stylectxt, GTK_STATE_FLAG_NORMAL, &color);
 
-          if (is_color_dark (&color))
-              color_extension = "-dark";
+          if (is_color_light (&color))
+              class_extension = "-dark";
 
```

Comment:
While absolutely correct, this got me confused ad first reading. Can you add a short comment explaining why a light color suggests a dark theme ? (I understand now this is because "color" is the text color, and if that's a light color that means the background is dark -> hence dark theme). We may have forgotten this logic in a few years time again...

And perhaps we should call the function "is_dark_theme" rather than "is_color_light" and then the comment can be inside that function. That would make it more clear to the caller what the purpose of the function is. Devs interested in how this is determined can the read the comment with the function.